lib/fetching/content-services/crowd-tangle-content-service.js

```diff
@@ -182,7 +182,7 @@ CrowdTangleContentService.prototype._parse = function(data, options) {
 	this.crowdtangle_saved_searches = crowdtangle_lists.crowdtangle_saved_searches;
 	// If the list is found and matched, then the ct_tag is the list name
 	if (this.crowdtangle_lists[data.account.id]) {
-		metadata.ct_tag = this.crowdtangle_lists[data.account.id].join(", ");
+		metadata.ct_tag = this.crowdtangle_lists[data.account.id];
```

Comment:
Does this mean that a post is only associated to 1 list? I think we want a post to have multiple lists. Post x can be in page Y that exists in two different lists. Hence, post x would also be in two different lists.

Reply:
No, this just means that the datastructure will be an array not a string, so if you're part of LIST_A and LIST_B, it'll look like `["LIST_A", "LIST_B"]` not `"LIST_A, LIST_B"`. I had to change the front end to accommodate this, but it lets us search by each list more naturally with the `$in` query rather than a regex search.

Reply:
cool